spec/runtime/utils/VoidInputStream.lsl

```diff
@@ -0,0 +1,22 @@
+//#! pragma: non-synthesizable
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "https://github.com/openjdk/jdk11/blob/master/src/java.base/share/classes/java/io/InputStream.java#L81";
+
+// imports
+
+import java/io/InputStream;
+
+
+// primary semantic types
+
+@GenerateMe
+@extends("java.io.InputStream")
+@public @final type VoidInputStream
```

Comment:
Why does this name have prefix "Void" ?

Reply:
*"Null"* usually means *"nothing"*. *"Void"*, on the other hand, is *"empty"*, if I'm not mistaken.
So, this is an *"empty input stream"*.